epub33/fxl-a11y-tech/index.html

```diff
@@ -61,8 +61,10 @@
             <h2>Introduction</h2>
         </section>
 
-        <section id="content-a11y">
-            <h2>Content Accessibility</h2>
+        <section id="content-a11y-programmatic">
+            <h2>Content Accessibility - Programmatic Techniques</h2>
+
+            <p>When building accessible content, there are two layers to consider: programmatic and visual. This section will cover programmatic techniques for content accessibility. Programmatic techniques are related to how the underlying code of the EPUB file is constructed. Features like reading order, alternative text for images, navigation, media overlays, and tables all have programmatic considerations.</p>
```

Comment:
The first sentence here could be used to build a longer explanation of the document's structure under Introduction title.